src/main/java/com/amazon/pvar/tspoc/merlin/utils/StackSafety.java

```diff
@@ -0,0 +1,29 @@
+package com.amazon.pvar.tspoc.merlin.utils;
```

Comment:
I don't see this file used anywhere (and not in the other pull requests you have open). Did you miss a file or is it coming later?

Reply:
Oh sorry, didn't mean to include it in this PR, will remove in the next revision.